app/routes/animals-tested.js

```diff
@@ -55,4 +57,6 @@ module.exports = [{
       }
     }
   }
-}]
+}
+
+module.exports = { handlers: [getHandler, postHandler] }
```

Comment:
Worth discussing this pattern a bit if we think we'll implement across multiple services. I think I'd prefer to just export and import the handlers explicitly. Wrapping them in a handlers array hides what you are getting a little bit.

So exporting as:
```
module.exports = { getAnimalsTested, postAnimalsTested }
```
Importing as:
```
const { getAnimalsTested, postAnimalsTested } = require('./routes/animals-tested')
```
Then consuming as:
```
server.route([
  getAnimalsTested,
  postAnimalsTested,
])
```
Everything very explicit and easy to follow through the code base.

Reply:
Yeah I'm down for that - do you think it makes sense to do it as part of this ticket, or when we move to ES modules? Because when we move ES, we wont be doing module.exports anymore and we could make the change then?

Main reason I'm asking is because this ticket was supposed to be timeboxed to a day.